uktrade/data-hub-frontend
# Refactor contact audit history to use new component

## Description of change

The contact audit history page has been refactored to use the new `AuditHistory` component introduced in #6552, meaning that there is now a consistent approach for edit/audit history pages.

## Test instructions

Go to the audit history page for a contact (`Interactionnnn Contactttt` in dev has a large history). In addition to the field names, you should now see the actual changes that were made.

## Screenshots

<details>
<summary>Before</summary>

![Screenshot 2024-03-01 at 13-04-09 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/21d00f15-5779-4fcd-8aed-9b5950f5822e)

</details>

<details>
<summary>After</summary>

![Screenshot 2024-03-01 at 13-04-28 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/cdd13642-f55d-4f82-817d-0e1b38938738)

</details>

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
